docs/email.md

```diff
@@ -0,0 +1,17 @@
+# Emails
+
+## Local Testing
+
+We use the gem [Letter Opener Web](https://github.com/fgrehm/letter_opener_web) to deliver emails in development for testing purposes. It provides a web interface based on the original [Letter Opener](https://github.com/ryanb/letter_opener) gem. All delivered emails can be accessed at `localhost:3000/letter_opener`.
+
+## Email Styles
+
+We have a mailer layout that supports both **Dark Mode** and **Light Mode** settings in the email client or based on system settings. The layout file [mailer.html.erb](../app/views/layouts/mailer.html.erb) has classes that get overriden depending on **light or dark mode** and will display some text, buttons and other components in appropriate colors.
+
```

Comment:
add screenshot of light and dark mode for an email. maybe the email confirmation one